src/tests/routes/me/tokens/snapshots/all__routes__me__tokens__create__create_token_success.snap

```diff
@@ -1,13 +1,14 @@
 ---
 source: src/tests/routes/me/tokens/create.rs
-expression: response.into_json()
+expression: response.json()
 ---
 {
   "api_token": {
     "crate_scopes": null,
     "created_at": "[datetime]",
     "endpoint_scopes": null,
     "expired_at": null,
+    "expiry_notification_at": null,
```

Comment:
do we want to expose this field on the public API? we can always add it later, but removing it would be tricky once we've first exposed it

Reply:
Sounds reasonable to me. We can make it private first. [`cc04749` (#8290)](https://github.com/rust-lang/crates.io/pull/8290/commits/cc047491b36a5a96dfbc25c8c90c9f00d2d2b59e)